src/components/backend-ai-environment-view.ts

```diff
@@ -113,15 +113,15 @@ export default class BackendAIEnvironmentView extends BackendAIPage {
         () => {
           this.is_superadmin = globalThis.backendaiclient.is_superadmin;
           this.isSupportContainerRegistryGraphQL =
-            globalThis.backendaiclient.supports('container-registry-gql');
+            globalThis.backendaiclient?.supports('container-registry-gql');
         },
         true,
       );
     } else {
       // already connected
       this.is_superadmin = globalThis.backendaiclient.is_superadmin;
       this.isSupportContainerRegistryGraphQL =
-        globalThis.backendaiclient.supports('container-registry-gql');
+        globalThis.backendaiclient?.supports('container-registry-gql');
```

Comment:
In this line, `?` is not required because it's already connected.
```suggestion
        globalThis.backendaiclient.supports('container-registry-gql');
```

Reply:
@yomybaby Logically you are right. However it is converted without predefine check routing after bundling.